src/AgentHeaderDescriptor.php

```diff
@@ -40,6 +40,7 @@
 class AgentHeaderDescriptor
 {
     const AGENT_HEADER_KEY = 'x-goog-api-client';
+    const GAX_VERSION = '0.5.0-dev';
```

Comment:
Probably add a TODO and created an issue here (similar to Java) that replace this hardcoded data with automatic detection.